main.go

```diff
@@ -56,6 +56,11 @@ func main() {
 			Value: 3000,
 			Usage: "libp2p connection manager 'high' water mark",
 		},
+		&cli.IntFlag{
+			Name:  "inmem-block-cache",
+			Value: 1 << 30,
+			Usage: "Size of the in-memory block cache. 0 to disable (disables compression too)",
+		},
```

Comment:
Total memory consumption would be something like block-cache + memtables + indexes + bloom filters + LSM.

Now, I am not sure how much of the LSM lives in memory in badger. They seem to suggest that the LSM can live in memory, and in V0 LSM's open option was set to MemoryMap, so very probably it depends on the OS  how many memory pages it keeps live.